### PR TITLE
fix: Align to the new atlasmap json format for java class .

### PR DESCRIPTION
**Pull request by iocanel**

### CODE REVIEW

File: inspector/src/test/resources/twitter4j.LoggerFactory.json [hunk 1]
```diff
@@ -1,32 +1,25 @@
...
+    "packageName" : "twitter4j",
+    "annotation" : false,
+    "annonymous" : false,
+    "enumeration" : false,
+    "localClass" : false,
+    "memberClass" : false,
+    "uri" : "atlas:java?className=twitter4j.LoggerFactory",
+    "interface" : false
+  }
+}
```
Comment (kahboom) [inline inspector/src/test/resources/twitter4j.LoggerFactory.json]: newline would be nice ;)

**Comment by rhuss** [comment]
@iocanel the pr-merge job is stil waiting to be running. Is this because of the pv hickup ?

**Comment by jimmidyson** [comment]
@rhuss @iocanel Let me see if killing and rerunning helps...

**Comment by jimmidyson** [comment]
oh dear there's lots of jobs waiting for something... https://jenkins-syndesis-ci.b6ff.rh-idev.openshiftapps.com/blue/organizations/jenkins/syndesis-rest/activity

**Comment by rhuss** [comment]
[merge]

**Pull request merged**